content/docs/2.9/scalers/selenium-grid-scaler.md

```diff
@@ -143,3 +143,47 @@ spec:
         browserName: 'chrome'
         browserVersion: '90.0'
 ```
+
+### Authentication Parameters
+
+The `url` parameter can optionally be specified via a `TriggerAuthentication` or `ClusterTriggerAuthentication` resource. This allows securely specifying a username and password in the URL when Selenium Grid's Basic HTTP Authentication is configured.
```

Comment:
Would you mind aligning the formatting with other scalers, for example https://github.com/kedacore/keda-docs/blob/main/content/docs/2.9/scalers/azure-storage-queue.md#authentication-parameters?

Reply:
Done. Let me know if this works or needs any other changes. Thanks!